src/registry/index.ts

```diff
@@ -148,6 +148,10 @@ enum MetaMacros {
   cssref = 'cssref',
   jsref = 'jsref',
   jssidebar = 'jssidebar',
+  glossarysidebar = 'glossarysidebar',
+  htmlsidebar = 'htmlsidebar',
+  svgref = 'svgref',
```

Comment:
the output of meta macros goes separately from other content. So, first:
- all sidebars should go to meta macros, otherwise they'll output directly into the page and it won't end up well.
- there also changes to be made on platform, because it also recognizes only a certain subset of macros

Reply:
So, yes: your change is absolutely necessary here, and also it's no sufficient to get the structure displayed on UI

Reply:
No worries, I debugged everything in https://github.com/webdoky/platform-next, just wasn't always sure I know what I'm doing